scripts/service-vuetorrent.sh

```
@@ -0,0 +1,248 @@
+  local service="$1"
+  case "$service" in
+    local_dns)
+      printf '%s' "arr_local_dns"
+      ;;
+    *)
+      printf '%s' "$service"
+      ;;
+  esac
+}
+
+declare -a ARR_STACK_PREVIOUS_RUNNING_SERVICES=()
```

Shellcheck (0.11.0)

[warning] ARR_STACK_PREVIOUS_RUNNING_SERVICES appears unused. Verify use (or export if used externally).

(SC2034)